include/nbl/ui/IInputEventChannel.h

```diff
@@ -107,19 +135,30 @@ class IMouseEventChannel : public impl::IEventChannelBase<SMouseEvent>
     virtual ~IMouseEventChannel() = default;
 
 public:
+    IMouseEventChannel(size_t circular_buffer_capacity) : base_t(circular_buffer_capacity)
```

Comment:
does `base_t` assert that the capacity is a power of two?

Reply:
Yeah, circular buffers  have those assertions